packages/host/config/environment.js

```diff
@@ -40,6 +40,8 @@ module.exports = function (environment) {
     loginMessageTimeoutMs: 1000,
     minSaveTaskDurationMs: 1000,
     iconsURL: process.env.ICONS_URL || 'https://boxel-icons.boxel.ai',
+    stripePaymentLink:
+      process.env.STRIPE_PAYMENT_LINK || 'https://stripe.example.com/payment',
```

Comment:
Don't put fictional urls here - even in development this link should lead to the real stripe payment link, either sandbox or test

Reply:
We will have a bigger set of stripe payment links, for example links to buy extra credits (links to buy 12500, 15000, 80000 credits). So adding all these links using env vars could become unwieldy. I am wondering what would be a good way to store these stripe payment links. Perhaps there could be an ENV var that is actually a JSON string. Or maybe a new db table that holds these links. I think this would be a good topic to discuss in the boxel meeting

Reply:
Also a link to Stripe Customer portal (where users can change or cancel subscription), we will need that too for each environment